### ISSUE TITLE
Brazilian states

### ISSUE THREAD

**Issue by linconkusunoki** [brazilian-utils/brazilian-utils#82]
What do you guys think about functions to handle the Brazilian states?

something like:
- Get all states
- Find a specific state

where this could be applied?
- Select a state
- Show a state by its code

```javascript
[
{code: 'sp', name: 'São Paulo'}, 
{code: 'rj', name: 'Rio de Janeiro'}
]
```

**Comment by hyanmandian** [comment]
Hi @linconkusunoki , awesome idea! We already have states info [here](https://github.com/brazilian-utils/brazilian-utils/blob/master/src/common/states.ts) (missing full state name, but we can add) and we just need to expose this. About "find a specific state", who uses this info should implement your search logic.

What do you think?

**Comment by linconkusunoki** [comment]
awesome @hyanmandian! It'll be very helpful to have the states available.
I think that makes sense only to expose the data and let the developers implement the logic :)

**Comment by nolleto** [comment]
Hey, I just create the PR with this method (`getStates`). Can you check if everything is alright? 😄

**Comment by hyanmandian** [comment]
Closed by #85
